Formula/ruby@1.9.3.rb

```diff
@@ -0,0 +1,180 @@
+class RubyAT193 < Formula
+  desc "Powerful, clean, object-oriented scripting language"
+  homepage "https://www.ruby-lang.org/"
+  url "https://cache.ruby-lang.org/pub/ruby/1.9/ruby-1.9.3-p551.tar.bz2"
+  sha256 "b0c5e37e3431d58613a160504b39542ec687d473de1d4da983dabcf3c5de771e"
+  revision 4
```

Comment:
Not so new since they have been introduced in homebrew-versions. We could even bump those revisions for each of them?